docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -469,7 +464,7 @@
     "therm_abs_rate = openmc.Tally(name='therm. abs. rate')\n",
     "therm_abs_rate.scores = ['absorption']\n",
     "therm_abs_rate.filters = [openmc.Filter(type='energy', bins=[0., 0.625e-6])]\n",
-    "tallies_file.add_tally(therm_abs_rate)"
+    "tallies_file.append(therm_abs_rate)"
```

Comment:
Jupyter won't load this notebook since it doesn't meet the JSON specs. I think this line may be missing a comma?